libmscore/shape.h

```diff
@@ -71,7 +71,7 @@ class Shape : public std::vector<ShapeElement> {
       qreal top() const;
       qreal bottom() const;
 
-      int size() const   { return std::vector<ShapeElement>::size(); }
+      int size() const   { return int(std::vector<ShapeElement>::size()); }
```

Comment:
I would use `size_t` instead as a return value.

Reply:
That was my first idea and attempt too, but that leads to tons of further warnings sat places where the result of these methods are used and compared with other values

Reply:
ouch... Let's add the //TODO comment here with the proposal to replace all the casts to `size_t`. Once, we have to do it.

Reply:
Maybe. OTOH these casts just do what the compiler implicitly does anyhow.

Reply:
Seems I was wrong, probably die to missing the tree in the forest, `size_t` is possible here